src/OrchardCore.Modules/OrchardCore.AuditTrail/Licence.md

```diff
@@ -0,0 +1,11 @@
+Copyright 2020, Finitive LLC
```

Comment:
This is a concern. Given we distribute OC with a BSD license, should we then restrict components to a different license with another copyright holder?

Reply:
Any contribution is going through the CLA. This is the related point:

> Originality of Work. You represent that each of Your Submissions is entirely Your original work. Should You wish to Submit materials that are not Your original work, You may Submit them separately to the Project if You (a) retain all copyright and license information that was in the materials as You received them, (b) in the description accompanying Your Submission, include the phrase "Submission containing materials of a third party:" followed by the names of the third party and any licenses or other restrictions of which You are aware, and (c) follow any other instructions in the Project’s written guidelines concerning Submissions.

https://cla.dotnetfoundation.org/OrchardCMS/OrchardCore?pullRequest=6679

Reply:
The module is open-source, uses the same BSD license, and is up here too: https://github.com/Finitive-LLC/Orchard-Core-Audit-Trail According to the module's license, it can be included in other projects if the license is included.

It was the request of Finitive to retain the copyright of the module but still let it be included in the Orchard source. This was the solution and I think it satisfies both of these completely. To be clear, I've also added this to the PR's description.

Reply:
Will you add any readme file with "Whatmodule do"?

Reply:
Yes, I will add a documentation for the module soon.

Reply:
Thanks for clarifying @Piedone

Reply:
> 
> 
> Yes, I will add a documentation for the module soon.

Thank you! Great Job!